### DISCUSSION TITLE
Stable Diffusion Gets A Major Boost With RTX Acceleration

### DISCUSSION THREAD

**Discussion by ghost** [lllyasviel/Fooocus#746] [Ideas]
https://www.nvidia.com/en-us/geforce/news/game-ready-driver-dlss-3-naraka-vermintide-rtx-vsr/
![image](https://github.com/lllyasviel/Fooocus/assets/8020186/d5e5a4ee-0a01-46a5-a0bc-0fbe14db3c1b)

https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT

**Comment by ghost** [comment]
From your base SD webui folder

Delete the venv folder

In the extensions folder delete: stable-diffusion-webui-tensorrt folder if it exists

Open a command prompt and navigate to the base SD webui folder

Run webui.bat - this should rebuild the virtual environment venv

When the WebUI appears close it and close the command prompt

Open a command prompt and navigate to the base SD webui folder

enter: venv\Scripts\activate.bat

the command line should now have (venv) shown at the beginning.

enter the following commands:

python.exe -m pip install --upgrade pip

python -m pip install nvidia-cudnn-cu11==8.9.4.25 --no-cache-dir

python -m pip install --pre --extra-index-url https://pypi.nvidia.com/ tensorrt==9.0.1.post11.dev4 --no-cache-dir

python -m pip uninstall -y nvidia-cudnn-cu11

venv\Scripts\deactivate.bat

webui.bat

Install the TensorRT extension using the Install from URL option

Once installed, go to the Extensions >> Installed tab and Apply and Restart

**Comment by Remember2015** [comment]
From [TersorRT](https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT):
------
"Native SDXL support coming in a future release. Please use the [dev branch](https://github.com/AUTOMATIC1111/stable-diffusion-webui/tree/dev) if you would like to use it today. Note that the Dev branch is not intended for production work and may break other things that you are currently using."

**Comment by barepixels** [comment]
From what I understand, you have to train new models for that.  I believe no models from Civitai will work.  Not sure about Lora.
